src/views/partials/breaches.js

```diff
@@ -36,7 +36,7 @@ function createBreachRows (data, logos) {
       const logo = getBreachLogo(breach, logos)
 
       return `
-      <details class='breach-row' data-status=${status} data-email=${account.email} data-classes='${dataClassesTranslated}' ${isHidden ? 'hidden' : ''}>
+      <details class='breach-row' data-company-name=${breach.Title} data-status=${status} data-email=${account.email} data-classes='${dataClassesTranslated}' ${isHidden ? 'hidden' : ''}>
```

Comment:
I'm a bit unclear on `data-company-name` vs `breach.Title` here.
Since this isn't really user-facing, do we want to use `breach.Title` (with potential spaces+punctuation) versus `breach.Name`?

`breach.Title` should always be unique, but not always "English" (if that matters in the pipeline):

```sh
…
{ Name: 'Zhenai', Title: 'Zhenai.com' }
{ Name: 'Zoosk', Title: 'Zoosk (2011)' }
{ Name: 'Zoosk2020', Title: 'Zoosk (2020)' }
{ Name: 'Parapa', Title: 'Пара Па' }
{ Name: 'SprashivaiRu', Title: 'Спрашивай.ру' }
{ Name: 'DecoratingTheHouse', Title: '집꾸미기' }
```

Reply:
Hm, I think you're right, but I'm a little on the fence since I think it'll be harder for the folks will need to figure out the mapping between the two, it won't match what's in the UI. I guess looking at the HTML will be a reliable way to do that on a one-off basis but that's not a great experience.

Reply:
I'm imagining product folks and data science folks being the primary consumer of the analytics and reports, to be specific. However I think the formatting consistency of the `Name` is a really good point.

Reply:
For the record, I don't know anything about anything&hellip; but&hellip; here's a free theory for you (since I saw the Zoosk and Zoosk2020 above). 

Back in 2011 Zoosk was breached. It wasn't reported/added to HIBP until 2017. Suspiciously, according to HIBP, that record was modified sometime in mid-2020 (3.5 years after it was originally added to the site?!).

<details>
<summary><a href="https://haveibeenpwned.com/api/v3/breach/zoosk">haveibeenpwned.com/api/v3/breach/zoosk</a></summary>

```js
Name: "Zoosk",
Title: "Zoosk (2011)",
Domain: "zoosk.com",
BreachDate: "2011-01-01",
AddedDate: "2017-02-08T07:59:39Z",
ModifiedDate: "2020-08-06T21:54:13Z",
```
</details>

My theory is that zoosk's `Title` was renamed from "Zoosk" to "Zoosk (2011)" when the new "Zoosk (2020)" breach was added to the site to help disambiguate the two breaches.

<details>
<summary><a href="https://haveibeenpwned.com/api/v3/breach/zoosk2020">haveibeenpwned.com/api/v3/breach/zoosk2020</a></summary>

```js
Name: "Zoosk2020",
Title: "Zoosk (2020)",
Domain: "zoosk.com",
BreachDate: "2020-01-12",
AddedDate: "2020-08-07T03:46:38Z",
ModifiedDate: "2020-08-07T03:46:38Z",
```
</details>

Not sure if any of that nonsense means anything, but it _might_ imply that `Name`s are immutable but `Title`s aren't. Unclear on whether that would impact product/data science folks and their queries, and all speculation on my part, but something to maybe consider (or mock).

We might be able to see if that's a valid theory by scraping all breaches and seeing how many map to the same `Domain` and then checking the `ModifiedDate`s to see if the theory is maybe correct.